cedar-policy/src/api/err.rs

```diff
@@ -161,17 +161,17 @@ impl From<cedar_policy_core::authorizer::ReauthorizationError> for Reauthorizati
     }
 }
 
-/// Errors serializing Schemas to the natural syntax
+/// Errors serializing Schemas to the Cedar syntax
 #[derive(Debug, Error, Diagnostic)]
-pub enum ToHumanSyntaxError {
+pub enum ToCedarSyntaxError {
```

Comment:
I think the name `ToCedarSchemaError` would better convey that this error is schema specific

Reply:
Done.